Cargo.toml

```diff
@@ -23,4 +23,4 @@ ark-serialize = { git = "https://github.com/Lightprotocol/algebra", branch="rele
 overflow-checks = true
 
 [profile.test]
-opt-level = 2
+opt-level = 0
```

Comment:
Just curious - why? Is it related to the stack memory issues?

Reply:
tried this for possible compiler bug